lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3578,6 +3573,12 @@ namespace {
           continue;
         }
 
+        // If we've already added this decl to the resulting struct, skip it so
+        // we don't add the same member twice.
+        if (!allMembers.insert(member).second) {
```

Comment:
We don't want to do this before L3570/3565 when we try to import the decl?

Reply:
Hmm yes, that makes sense. Fixed.